src/atlas_f/functionspace/atlas_functionspace_CellColumns_module.fypp

```diff
@@ -0,0 +1,286 @@
+! (C) Copyright 2013 ECMWF.
+!
+! This software is licensed under the terms of the Apache Licence Version 2.0
+! which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+! In applying this licence, ECMWF does not waive the privileges and immunities
+! granted to it by virtue of its status as an intergovernmental organisation nor
+! does it submit to any jurisdiction.
+
+#include "atlas/atlas_f.h"
+#:include "internals/atlas_generics.fypp"
+
+#:set ranks = [0,1]
```

Comment:
Please remove, and also rename file with ".F90" extension. No fypp is used here.

Reply:
Thanks! Done.